pyuvdata/uvh5.py

```diff
@@ -13,6 +13,12 @@
 from .uvdata import UVData
 from . import utils as uvutils
 
+try:
+    import h5py
```

Comment:
The gentler error messages are much improved! But I think this will generate an error and not allow the package to be installed if the user doesn't have h5py. The reason is that in the `__init.py__`, we have `from .uvh5 import *`, which would trigger this import error, but we don't have an analogous import statement for `ms.py`. I think it's easiest just to remove the uvh5 import from the init file, which would just have the end-user impact of not being able to do something like `from pyuvdata import UVH5` to get uvh5 objects directly, which I don't think is a very common use case. If they really wanted to, they could always do `from pyuvdata.uvh5 import UVH5`. (I think this is correct, based on my understanding of how python packaging works.)

Reply:
I think you're right, but I also think taking out the import in the `__init__` is pretty reasonable, although maybe we should check with our users. I don't think people normally import the file-specific objects and maybe they should all come out of the `__init__`...